cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/servlet/UserTaskManagerTest.java

```diff
@@ -71,6 +71,22 @@ public void testCreateUserTask() {
     Assert.assertEquals(userTaskHeaderValue.getValue(), testUserTaskId.toString());
     Assert.assertEquals(future, future3);
 
+    EasyMock.reset(mockHttpServletResponse);
+    // test-case: for sync task, UserTaskManager does not create mapping between request URL and UUID.
+    HttpServletRequest mockHttpServletRequest3 = prepareRequest(null, null, "test_sync_request", Collections.emptyMap());
+    userTaskManager.getOrCreateUserTask(mockHttpServletRequest3, mockHttpServletResponse, uuid -> future, 0, false).get(0);
+
+    // mockHttpServletRequest3 does not contain User-task-id, so the only place to retrieve non-null user-task-id is _sessionKeyToUserTaskIdMap.
```

Comment:
Why do we use inconsistent names such as `user-task-id` and `User-task-id` to represent `UserTaskManager#USER_TASK_HEADER_NAME`? Would it be possible to say something like "If the request header does not contain a valid user task id..."?

Reply:
will change that.